packages/widgets/src/health-monitoring/component.tsx

```diff
@@ -227,9 +266,14 @@ export default function HealthMonitoringWidget({ options, integrationIds }: Widg
 }
 
 export const formatUptime = (uptimeInSeconds: number, t: TranslationFunction) => {
-  const days = Math.floor(uptimeInSeconds / (60 * 60 * 24));
-  const remainingHours = Math.floor((uptimeInSeconds % (60 * 60 * 24)) / 3600);
-  return t("widget.healthMonitoring.popover.uptime", { days, hours: remainingHours });
+  dayjs.extend(duration);
+  const uptimeDuration = dayjs.duration(uptimeInSeconds, "seconds");
+  const days = uptimeDuration.days();
+  const hours = uptimeDuration.hours();
+  const minutes = uptimeDuration.minutes();
+  const formattedUptime = `${days} ${t("common.information.days")}, ${hours} ${t("common.information.hours")}, ${minutes} ${t("common.information.minutes")}`;
```

Comment:
Either revert this as it's a lot more infos or improve it so that it can also work in other languages